nuxeo-ai-model/src/main/java/org/nuxeo/ai/model/AiDocumentTypeConstants.java

```diff
@@ -19,40 +19,40 @@
 package org.nuxeo.ai.model;
 
 /**
- * Simple class to store constants related to AI Document Types: Ai_Model and Ai_Corpus
+ * Simple class to store constants related to AI Document Types: Ai_Model and DatasetExport
  */
 public class AiDocumentTypeConstants {
 
     // corpus type constants
-    public static final String CORPUS_TYPE = "AI_Corpus";
```

Comment:
If this is used for cloud upload then it shouldn't be changed.

Reply:
don't see any usage.